app/views/admin/users-id.html

```diff
@@ -167,9 +167,12 @@ <h1 ng-show="document.UserID">Update a user profile</h1>
 						<div class="row">
 							<div class="col-sm-12">
 								<div class="form-group">
-									<label>Government</label>
-									<div><select class="form-control" ng-model="document.Government" ng-options="c.code as c.name for c in countries"></select>
-									<span  class="btn btn-default btn-xs pull-right" ng-show="document.Government"  ng-click="document.Government = undefined;"> clear government </span></div>
+									<div ng-class="{'alert alert-danger' : !canGrantGovernment()}">
+										<label>Government</label>
+										<div><select class="form-control" ng-disabled="!canGrantGovernment()" ng-model="document.Government" ng-options="c.code as c.name for c in countries"></select>
```

Comment:
We should keep the field enabled if a value is set even if admin||scbd so we can remove the value. Or we 
- add a button to unset
-  prevent adding admin || scbd if user gov already set
- Auto unset